ta_services/secure_key_services/src/processing_asymm.c

```diff
@@ -138,6 +139,11 @@ static uint32_t sks2tee_algorithm(uint32_t *tee_id,
 		break;
 	}
 
+	if (*tee_id == TEE_ALG_RSAES_PKCS1_V1_5 &&
+		(function == SKS_FUNCTION_SIGN ||
+		 function == SKS_FUNCTION_VERIFY))
+		*tee_id = TEE_ALG_RSASSA_PKCS1_V1_5;
```

Comment:
This is part of the function that translates the SKS_CKM_* to tee_id, and SKS_CKM_RSA_PKCS can be either TEE_ALG_RSAES_PKCS1_V1_5 or TEE_ALG_RSASSA_PKCS1_V1_5 based on whether it is called with C_Encrypt/C_Decrypt or C_Sign/C_Verify. I will place this chunk closer to the code that does the initial translation, that way it will be more intuitive.

Reply:
ok, fine. I did not understand the sequence at the time.